src/main/java/easy/mind/com/api/сontroller/TestsQuestionsController.java

```diff
@@ -0,0 +1,31 @@
+package easy.mind.com.api.сontroller;
+
+import easy.mind.com.api.DTO.TestsQuestionsDTO;
+import easy.mind.com.api.entity.TestsQuestions;
+import easy.mind.com.api.facade.TestsQuestionsFacade;
+import easy.mind.com.api.service.TestsQuestionsService;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users/{userId}/questions")
+@RequiredArgsConstructor
+public class TestsQuestionsController {
+    private final TestsQuestionsFacade testsQuestionsFacade;
+
+    @GetMapping("/{id}")
+    public ResponseEntity<TestsQuestionsDTO> read(@PathVariable int id,  @PathVariable int userId) {
```

Comment:
`userId` is not used here. Should this method be in this controller?

Reply:
This and `getAllQuestions` methods seems to be not related to user anyhow. Shouldn't they be moved to other controller?

Reply:
it is needed here, because on this ID we will check whether the user has not passed this test in the last 24 hours

Reply:
So that check will be added in the future? Okay they